src/onboarding/steps.ts

```diff
@@ -63,12 +63,13 @@ export function onboardingPropsSelector(state: RootState): OnboardingProps {
  * and count the number of screens until the given screen and the total number
  */
 export function getOnboardingStepValues(screen: Screens, onboardingProps: OnboardingProps) {
-  let stepCounter = 1
+  let stepCounter = 1 // will increment this up to the onboarding step the user is on
   let totalCounter = 1
-  let reachedScreeen = false
-  let currentScreen: Screens = FIRST_ONBOARDING_SCREEN
+  let reachedScreeen = false // tracks whether we have reached the screen the user is on in onboarding, and we can stop incrementing stepCounter
```

Comment:
```suggestion
  let reachedScreeen = false // tracks whether we have reached the step the user is on in onboarding, and we can stop incrementing stepCounter
```
since you clarified the screens vs steps thing, I realized this should be "step", not "screen"